lib/lighthouse-reporter.js

```diff
@@ -134,10 +134,13 @@ async function writeReport(url, flags = {}, defaultChromeFlags = [], lighthouseF
   const { chromeFlags, configPath, budgetPath, ...extraLHFlags } = lighthouseFlags;
   const customChromeFlags = chromeFlags ? chromeFlags.split(',') : [];
 
-  const lighthouseResult = await launchChromeAndRunLighthouse(url, extraLHFlags, configPath, budgetPath, [
-    ...defaultChromeFlags,
-    ...customChromeFlags,
-  ]);
+  const lighthouseResult = await launchChromeAndRunLighthouse(
+    url,
+    [...defaultChromeFlags, ...customChromeFlags],
+    extraLHFlags,
+    configPath,
+    budgetPath,
+  );
```

Comment:
This change comes from the pull request https://github.com/andreasonny83/lighthouse-ci/pull/71, and will be discarded as soon as the previous PR is merged (and this one rebased).